internal/config/app_config_sort.go

```diff
@@ -63,6 +65,10 @@ func (c *genericConfig) sortVars(ctx *hcl.EvalContext) ([]*analyzedPair, error)
 	}
 
 	for _, m := range maps {
+		if m.expr == nil {
```

Comment:
Quick comment: why/when does this happen?

Reply:
Cleanup the logic, it was only for a test but I just modified the tests to not require it.